code/client/makecatalogs

```diff
@@ -95,7 +95,10 @@ def main():
         exit(-1)
 
     # Make the catalogs
-    errors = makecatalogslib.makecatalogs(repo, options, output_fn=print)
+    try:
+        errors = repo.makecatalogs()
```

Comment:
Do we want to pass along the same parameters to `repo.makecatalogs()`? The implementation would be free to ignore them, but perhaps they might be useful.

Reply:
Good point. For my uses, they aren't needed, but make sense for some future proofing. Updating...